lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/implementation/Subplan.java

```diff
@@ -23,9 +23,10 @@
 import java.util.*;
 
 public final class Subplan<T> extends InjectionPlan<T> {
-  final InjectionPlan<? extends T>[] alternatives;
-  final int numAlternatives;
-  final int selectedIndex;
+
+  private final InjectionPlan<? extends T>[] alternatives;
+  private final int numAlternatives;
+  private final int selectedIndex;
```

Comment:
These visibility changes look irrelevant. Shall we make another PR for these visibility changes?

Reply:
Thanks for looking at it, @dongjoon-hyun ! This change is also related to CheckStyle - it complained that package-level scope for these member variables was too broad, so I've tightened it up. We can move that change into a separate PR, but then we'll have CheckStyle warnings in this one

Reply:
Then, please make a separate PR and merge that first before this PR. I can help that, too.